lib/ui/painting.dart

```diff
@@ -1023,6 +1023,48 @@ enum Clip {
 // constant and can't propagate into the set/get calls.
 const Endian _kFakeHostEndian = Endian.little;
 
+/// An immutable image dimensions container that holds the target dimensions.
```

Comment:
How is this not identical to `Size`?

Reply:
Size passes dimensions in `double` not `int`. I was trying to avoid casts / rounding when converting them for skia.